[PAY-1367][PAY-1400][PAY-1401][PAY-1402][PAY-1419] Fix chat tile unfurl bugs

sddioulde opened the pull request:
### Description

- unfurl albums
- remove 1px bottom border on unfurls with no text
- don't scale up/down on tile hover/active
- fix border radius on the chat tiles
- pause chat playlist on mobile when pressed while it's playing

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

local web and mobile dapps v stage

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

audius-infra commented:
Preview this change https://demo.audius.co/sd-chat-unfurl-bugs

audius-infra commented:
Preview this change https://demo.audius.co/sd-chat-unfurl-bugs

audius-infra commented:
Preview this change https://demo.audius.co/sd-chat-unfurl-bugs

audius-infra commented:
Preview this change https://demo.audius.co/sd-chat-unfurl-bugs

audius-infra commented:
Preview this change https://demo.audius.co/sd-chat-unfurl-bugs

audius-infra commented:
Preview this change https://demo.audius.co/sd-chat-unfurl-bugs

dharit-tan commented on packages/mobile/src/screens/chat-screen/ChatMessageListItem.tsx:
```diff
@@ -222,23 +217,34 @@ export const ChatMessageListItem = memo(function ChatMessageListItem(
-      ? { ...styles.unfurl, ...styles.unfurlAuthor }
-      : { ...styles.unfurl, ...styles.unfurlOtherUser }
-    : styles.unfurl
+  const { secondaryDark1, neutralLight7 } = useThemeColors()
```
nit: separate this out into its own function?

dharit-tan commented on packages/mobile/src/screens/chat-screen/ChatMessageListItem.tsx:
```diff
@@ -222,23 +217,34 @@ export const ChatMessageListItem = memo(function ChatMessageListItem(
+  const { secondaryDark1, neutralLight7 } = useThemeColors()
+
+  const borderBottomColor = isAuthor ? secondaryDark1 : neutralLight7
+  const borderBottomWidth = hideMessage
```
what's the difference between borderWidth 0 vs undefined?

rickyrombo commented on packages/web/src/pages/chat-page/components/ChatMessageListItem.tsx:
```diff
@@ -178,6 +178,11 @@ export const ChatMessageListItem = (props: ChatMessageListItemProps) => {
     )
   }
 
+  const unfurlStyles = {
```
nit: can call cn here and set it to eg `unfurlClassNames` and pass to className directly below

dharit-tan commented on packages/mobile/src/screens/chat-screen/ChatMessageListItem.tsx:
```diff
@@ -276,21 +282,21 @@ export const ChatMessageListItem = memo(function ChatMessageListItem(
                 >
-                  {isPlaylistUrl(linkValue) ? (
+                  {isCollection ? (
                     <ChatMessagePlaylist
```
dang @rickyrombo these components are starting to look mighty similar lol. probs not worth consolidating at this point tho. if it ain't broke don't fix.

dharit-tan commented on packages/mobile/src/screens/chat-screen/ChatMessagePlaylist.tsx:
```diff
@@ -106,7 +106,7 @@ export const ChatMessagePlaylist = ({
   }, [tracks, uidMap])
 
-  const isActive = playingUid !== null && playingUid === uid
+  const isActive = tracksWithUids.find((track) => track.uid === playingUid)
```
nice

dharit-tan commented on packages/web/src/pages/chat-page/components/ChatMessagePlaylist.tsx:
```diff
@@ -129,25 +129,24 @@ export const ChatMessagePlaylist = ({
-        variant='readonly'
-      />
-    </div>
+    // You may wonder why we use the mobile web playlist tile here.
```
lol

rickyrombo reviewed (commented):
I may have an idea on how to simplify this a bit - mind if I branch off and make a PR to your PR?

There's still a few bugs with your impl from what I can see:

Web unfurls by themselves have pink undertones to their corners, and aren't rounded on the bottom left:
![image](https://github.com/AudiusProject/audius-client/assets/3690498/2d41bcae-6691-40b7-a3b9-1aadab456b01)

Web image urls are a bit busted:
![image](https://github.com/AudiusProject/audius-client/assets/3690498/aa060d80-96cf-4751-a7a2-3568bd67f00d)

sddioulde commented:
as im off today, i quickly addressed some of the comments, and im merging so that the changes are available before next QA on monday. looks like the remaining items are mostly nits too.
@dharit-tan feel free to make a new branch from main to address the web image url bug you're talking about. thx
